teps/0044-data-locality-and-pod-overhead-in-pipelines.md

```diff
@@ -154,8 +154,8 @@ We can explore providing this functionality in a later iteration of this proposa
   can be run on the same pod as another Task that fails
   - This is to support use cases such as uploading test outputs, even if the test Task failed
   - This requirement is being included because we could choose a solution that doesn't address the above use case.
-1. Any configuration for the execution of a Pipeline must be modifiable at runtime.
-  - We may explore adding authoring time configuration in the future after gathering feedback on runtime configuration.
+1. The chosen solution should allow us to provide both authoring time and runtime configuration to colocate multiple Tasks.
+  - We will start with only configuration for one of (authoring time, runtime) and optionally provide the other based on user feedback.
```

Comment:
sounds good - though would prefer that we make this decision (authoring time vs runtime) before marking this TEP implementable